webui/api-ts-sdk/Makefile

```diff
@@ -7,7 +7,7 @@ all: get-deps build
 .PHONY: get-deps-swagger
 get-deps-swagger:
 	mkdir -p deps
-	curl https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.14/swagger-codegen-cli-2.4.14.jar \
+	[ -f deps/swagger-codegen-cli.jar ] || curl https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.14/swagger-codegen-cli-2.4.14.jar \
```

Comment:
nice, no need to re-download this every time. Just need to run `make clean` if we ever need to get a new version which shouldn't happen too often.

Reply:
> Just need to run make clean if we ever need to get a new version which shouldn't happen too often.

exactly. I ended up addressing this in a better way covering this case as well and put up a separate PR https://github.com/determined-ai/determined/pull/1838  so I'll take it out of here.